source/change-streams/tests/change-streams.yml

```diff
@@ -297,3 +297,163 @@ tests:
+        name: updateOne
+        arguments:
+          filter:
+            x: 1
+          update:
+            $set:
+              x: 2
+      -
+        database: *database_name
+        collection: *collection_name
+        name: replaceOne
+        arguments:
+          filter:
+            x: 2
+          replacement:
+            x: 3
+      -
+        database: *database_name
+        collection: *collection_name
+        name: deleteOne
+        arguments:
+          filter:
+            x: 3
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              -
+                $changeStream:
+                  fullDocument: default
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success:
+        -
+          operationType: insert
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            x:
+              $numberInt: "1"
+        -
+          operationType: update
+          ns:
+            db: *database_name
+            coll: *collection_name
+          updateDescription:
+            updatedFields:
+              x:
+                $numberInt: "2"
+        -
+          operationType: replace
+          ns:
+            db: *database_name
+            coll: *collection_name
+          fullDocument:
+            x:
+              $numberInt: "3"
+        -
+          operationType: delete
+          ns:
+            db: *database_name
+            coll: *collection_name
+  -
+    description: Test rename and invalidate event types
+    minServerVersion: "4.0.1"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline: []
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: rename
+        arguments:
+          to: *collection2_name
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            cursor: {}
+            pipeline:
+              -
+                $changeStream:
+                  fullDocument: default
+          command_name: aggregate
+          database_name: *database_name
+    result:
+      success:
+        -
+          operationType: rename
+          ns:
+            db: *database_name
+            coll: *collection_name
+          to:
+            db: *database_name
+            coll: *collection2_name
+        -
+          operationType: invalidate
+  -
+    description: Test drop and invalidate event types
+    minServerVersion: "4.0.1"
+    target: collection
+    topology:
+      - replicaset
+    changeStreamPipeline: []
+    changeStreamOptions: {}
+    operations:
+      -
+        database: *database_name
+        collection: *collection_name
+        name: drop
+        arguments:
```

Comment:
I saw blank YAML values being used in some transaction tests, which evaluates to `null`; however, I'd propose using explicit `~` or `null` here if the key is needed for clarity.

That said, can we just make `arguments` optional? It really has no purpose for `drop`, but I assume this may be the first operation in the test suite where we're calling a method with no arguments.